scripts/limonade/src/index.ts

```diff
@@ -100,55 +101,67 @@ async function run() {
       );
 
     console.log("Resubmitting opportunities", payloads.length);
-    for (const payload of payloads) {
-      try {
-        await client.submitOpportunity(payload);
-      } catch (e) {
-        console.error(e);
-      }
-    }
+    await Promise.all(
```

Comment:
there should be some batching mechanism here. It's not practical to send 1000 requests at the same time + we will probably get rate-limited on prod